src/credentials/GroupDataProviderImpl.cpp

```diff
@@ -799,11 +799,98 @@ CHIP_ERROR GroupDataProviderImpl::RemoveAllGroupMappings(chip::FabricIndex fabri
     }
 }
 
+GroupDataProvider::GroupMappingIterator * GroupDataProviderImpl::IterateGroupMappings(chip::FabricIndex fabric_index)
+{
+    VerifyOrReturnError(mInitialized, nullptr);
+    return mAllGroupsIterators.CreateObject(*this, fabric_index);
+}
+
+GroupDataProviderImpl::AllGroupMappingsIteratorImpl::AllGroupMappingsIteratorImpl(GroupDataProviderImpl & provider,
+                                                                                  chip::FabricIndex fabric_index) :
+    mProvider(provider),
+    mFabric(fabric_index)
+{
+    FabricData fabric(fabric_index);
+    ReturnOnFailure(fabric.Load(provider.mStorage));
+
+    // Existing fabric
+    mFirstEndpoint = fabric.first_endpoint;
+    mFirstGroup    = true;
+    mEndpointCount = fabric.endpoint_count;
+    mEndpoint      = fabric.first_endpoint;
+    mEndpointIndex = 0;
+}
+
+size_t GroupDataProviderImpl::AllGroupMappingsIteratorImpl::Count()
+{
+    // chip::EndpointId endpoint_id = mFirstEndpoint;
```

Comment:
Any reason this is commented? Seems like it could be removed

Reply:
Copy-and-paste leftover. Removed.